site/public/css/autograding-status.css

```diff
@@ -0,0 +1,22 @@
+.right-boarder {
+    border-right: 1px solid var(--text-black);
+}
+
+#autograding-status-table {
+    border: 1px solid var(--text-black);
+}
+
+#course-table > *,
+#machine-table > * {
+    text-align: left;
+}
+
+#toggle-btn {
+    margin: 1em 0 1em 0;
+}
+
+#course-table td:nth-of-type(1)::before { content: "Semester"; }
```

Comment:
What purpose are these trying to serve?  It seems to me like we're writing HTML in a CSS file here...

Reply:
This allows the table to be displayed properly when the windows become too small. Without these lines, the table displayed will consist of `f21`, `sample`, `grading_homework`, and so on without any of the column names,
![image](https://user-images.githubusercontent.com/49821332/129625341-8d0c42b9-0973-486b-9e83-9bdd4ec12df5.png)

Reply:
Oh, ok, sounds good then.  Might want to leave a comment near the table saying that this has been done, just in case someone comes along later and wants to edit this table.